distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/Emotion,MIT License.txt

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
JSON in txt file

Reply:
This was the result of no license file existing in the repo. The others are not intentional, and I'll have a look at them.

Reply:
You can probably use https://raw.githubusercontent.com/emotion-js/emotion/master/LICENSE